cmake/SigslotUtils.cmake

```diff
@@ -33,12 +33,7 @@ target_compile_options(Sigslot_CommonWarnings INTERFACE
 
 add_library(Sigslot_ManyWarnings INTERFACE)
 target_compile_options(Sigslot_ManyWarnings INTERFACE
-    $<$<BOOL:${SIGSLOT_COMPILER_CLANG_OR_CLANGCL_OR_GCC}>:
-        -Weverything;-Wno-c++98-compat;-Wno-c++98-compat-pedantic;-Wno-unused-macros;
-        -Wno-newline-eof;-Wno-exit-time-destructors;-Wno-global-constructors;
-        -Wno-gnu-zero-variadic-macro-arguments;-Wno-documentation;-Wno-missing-prototypes;
-        -Wno-padded;-Wno-weak-vtables;-Wno-c++17-extensions;-Wno-shadow-field-in-constructor;
-        -Wno-return-std-move-in-c++11>
+    $<$<BOOL:${SIGSLOT_COMPILER_CLANG_OR_CLANGCL_OR_GCC}>:-Wpedantic>
```

Comment:
that seems better to me, and many of this _-Wno-warnings_ are not known by `Apple clang compiler`!